docs/src/languages/php/tuning.md

```diff
@@ -8,211 +8,186 @@ Platform.sh offers a wide degree of flexibility in how PHP behaves,
 but that does mean you may need to take a few steps to ensure your site is running optimally.
 
 The following recommendations are guidelines only.
-They're also listed in approximately the order we recommend investigating them,
-although your mileage may vary.
+They're also listed in about the order to investigate them.
```

Comment:
:clipboard: This is one case where I'd say `about` doesn't really work as a substitution for `approximately`.

```suggestion
They're also listed in approximately the order to investigate them.
```